[SDK-1988] Update ASPNET Core 2.1 MVC Quickstart (#9474)
auth0/docs: articles/quickstart/webapp/aspnet-core/_includes/_login.md

}
```

::: note

> The `ConfigureSameSiteNoneCookies` method used above was added as part of the [sample application](https://github.com/auth0-samples/auth0-aspnetcore-mvc-samples/blob/netcore2.1/Quickstart/01-Login/Support/SameSiteServiceCollectionExtensions.cs) in order to make cookies with SameSite=None work over HTTP when using Chrome ([read more here](https://blog.chromium.org/2019/10/developers-get-ready-for-new.html)). We recommend using HTTPS instead of HTTP, which removes the need for the `ConfigureSameSiteNoneCookies` method.

Reviewer: I'd link some of the text that already exists rather than adding something like "read more", gives the reader more expectation about what they're about to click on. For screen readers, too.

```suggestion
The `ConfigureSameSiteNoneCookies` method used above was added as part of the [sample application](https://github.com/auth0-samples/auth0-aspnetcore-mvc-samples/blob/netcore2.1/Quickstart/01-Login/Support/SameSiteServiceCollectionExtensions.cs) in order to [make cookies with SameSite=None work over HTTP when using Chrome](https://blog.chromium.org/2019/10/developers-get-ready-for-new.html). We recommend using HTTPS instead of HTTP, which removes the need for the `ConfigureSameSiteNoneCookies` method.
```

Author: I made the exact change in the ASPNET Core 3.1 quickstart as part of this PR.

}
```

> ## Trigger Authentication

Author: Moved the order here to be in line with https://auth0.com/docs/quickstart/webapp/aspnet-core-3, which talks about Triggering Authentication before Calling an API.

}
```

> To retrieve the tokens, you can call `GetTokenAsync` and use it as needed:

Reviewer: ```suggestion
To retrieve the tokens, call `GetTokenAsync` and use them as required:
```

Author: I made the exact change in the ASPNET Core 3.1 quickstart as part of this PR.
